### PR TITLE
Route to make comment on task

### PR DESCRIPTION
**Pull request by AlaboGift**
I created Route to make comment on task

### CODE REVIEW

File: routes/api.php [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 // comment post request
 Route::post('comment', [\App\Http\Controllers\TaskCommentController::class, 'store']);
-Route::post('comment', [\App\Http\Controllers\CommentController::class, 'store']);
+Route::post('comments', [\App\Http\Controllers\CommentController::class, 'store']);
```
Comment (EricOgie) [inline routes/api.php:29]: Adding comment to a task is meant to be with respect to a specific task. I don't see how you are targeting the specific task that you want to add comments to.

Comment (AlaboGift) [reply routes/api.php:29]: The post contains the user Id as well as the task id, the user clicks on the task he wants to comment on and in the URL you have taskid/comment this taskid is sent with the post data

File: app/Http/Controllers/commentController.php [hunk 2]
```diff
@@ -2,23 +2,49 @@
...
+    public function index()
+    {
+    }
 
+    public function store(Request $request)
+    {
 
-    public function update($cid)
+        $id = rand(1, 20);
+        $userId = rand(1, 20);
```
Comment (EricOgie) [inline app/Http/Controllers/commentController.php:26]: You can not use a random id to identify a user that is meant to be in a system.

Comment (AlaboGift) [reply app/Http/Controllers/commentController.php:26]: Am not the one that is using the update function or wrote it

**Comment by EricOgie** [comment]
First,- there is a conflict issue in this branch.

2nd- Your route to add comments to a task should target the task you want to comment on.
        - The route should also include a way to identify the user commenting
        - Using a random id generator for a user that is meant to be in a system is never going to target the intended user.

**Pull request closed without merging**